core/src/ast/function.rs

```diff
@@ -1176,14 +1178,6 @@ mod tests {
             ))))
             .to_sql()
         );
-
-        assert_eq!(
```

Comment:
this removal does not look intentional which is not related to this pr.

Reply:
I fixed :)
>associated commit: 7d2eeb8